src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -473,7 +471,24 @@ public static GitChange GetChangeOpt(string path, string originalText, string ne
             }
         }
 
-        public static string GetBuildDescription(this Build build)
+        private static string CreatePullRequestDescription(Build oldBuild, Build newBuild, bool useMarkdown)
```

Comment:
>newBuild [](start = 81, length = 8)

Can we either rename newBuild to buildToInsert or the other way around? They should be referring to the same build always so we should keep the naming consistent